Automattic/wp-calypso
# Mobile web: Improve the way we show email address in site setup

## Before:

In the site setup checklist on the mobile layout under `My Home` we do not see the entered email address under the `Confirm your email address` step:

![](https://d.pr/i/GyGsgo+)
[Direct Link](https://d.pr/i/GyGsgo+)


### After:

![](https://d.pr/i/6FXoRo+)
[Direct Link](https://d.pr/i/6FXoRo+)

We can here see the email address which is viewable without the user needing to click on the `Confirm your email address` step.

#### Testing instructions

1. Create a new WordPress.com site with a new account
2. Goto https://wordpress.com/home/:site on a mobile layout to see the site setup checklist
3. We should now see the user email under the `Confirm your Email address` step without it being collapsed

![](https://d.pr/i/txD5UG+)


Related to #58360

@tyxla  @enejb
